packages/design-system/Header.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
```

Comment:
React import is not required for React 18. Remove this and any other cases across all files. Add doc comments too.